sample-bchd.conf

```diff
@@ -238,10 +266,50 @@
 ; notls=1
 
 
+; ------------------------------------------------------------------------------
+; Database Settings - The following options control the database that holds
+; the block chain.
+; ------------------------------------------------------------------------------
+
+; Database backend to use for the Block Chain.
+; Valid drivers: ffldb, test
```

Comment:
These are the drivers I saw are currently being registered. Does it make sense to list `test` here?

Reply:
I think this one could probably be removed. I don't think anyone would be changing this for anything other than tests. In which case they wouldn't be using a config file.